src/examples/tags/TagsClose.tsx

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Span } from '@zendeskgarden/react-typography';
+import { Tag } from '@zendeskgarden/react-tags';
+import { Row, Col } from '@zendeskgarden/react-grid';
+
+const Example = () => (
+  <Row>
+    <Col textAlign="center">
+      <Tag tabIndex={0}>
```

Comment:
Would it be helpful to highlight the use of `tabIndex` in the example's description?